Jint/Runtime/PromiseRejectedException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Jint.Native;
+
+namespace Jint.Runtime
+{
+    [Serializable]
```

Comment:
other exceptions in Jint do not have Serializable, so probably not needed. when SerializableAttribute should have serialization constructors.

Reply:
Sorry this was left over from Resharper's template! Removed.